py/datastore_service.py

```diff
@@ -103,11 +104,44 @@ def handle_request(self, post):
     #get the reporter data
     reports = self.parse_reports(post)
 
-    #TODO: build a query
-    #for report in reports:
+    try:
+      # check and see if prepared statement exists...if not, create it
```

Comment:
yeah we can do that....will have to be per connection due to threads i believe.   Should be easy to do.